web/conf/logback.xml

```diff
@@ -33,9 +29,6 @@
     <appender-ref ref="STDOUT" />
   </appender>
 
-  <logger name="play" level="INFO" />
-  <logger name="application" level="INFO" />
-
```

Comment:
I don't understand why this removes the loggers. Shouldn't the loggers use the new rolling appender somehow?

Reply:
I think we don't use any of these loggers. We just use  the root logger. Or did I miss any use?

Reply:
Ack. Reintroduced with [92f57a0](https://github.com/hbz/lobid-resources/pull/1919/commits/92f57a08eb52184a4f93f097f7765077fb13a243).